tests/tests/src/lampo_cln_tests.rs

```diff
@@ -539,3 +540,373 @@ fn be_able_to_kesend_payments() {
     assert!(result.is_ok(), "{:?}", result);
     async_run!(cln.stop()).unwrap();
 }
+
+#[test]
+fn test_closing_two_channels_without_channelid_success() {
```

Comment:
```suggestion
fn test_closing_two_channels_without_channelid_fails() {
```

It is returning an error right?